### ISSUE TITLE
pandas failed to import output

### ISSUE THREAD

**Issue by carlowahlstedt** [aws/sagemaker-python-sdk#2166]
I'm seeing this issue: https://github.com/aws/sagemaker-python-sdk/issues/1696

To reproduce:
```dockerfile
FROM python:3 as base

RUN pip install sagemaker

ENV AWS_DEFAULT_REGION us-east-2
WORKDIR /usr/src/app
COPY test_container.py .

CMD [ "python", "./test_container.py" ]
```
where `test_container.py` is:
```python
import sagemaker
```

**Comment by ahsan-z-khan** [comment]
Hi @carlowahlstedt ,

Thanks for using SageMaker. Can you please screenshots or logs to help explain your problem?

**Comment by carlowahlstedt** [comment]
Here's sample code: [sagemaker.zip](https://github.com/aws/sagemaker-python-sdk/files/6029177/sagemaker.zip)

If you run: `docker build -t sagemaker-test .`
Then run: `docker run -it --rm pagemaker-test`

You will see the error.
<img width="590" alt="image" src="https://user-images.githubusercontent.com/2687687/108851340-507e1200-75b2-11eb-94b4-2c6d7b97220f.png">

**Comment by ChoiByungWook** [comment]
> Here's sample code: [sagemaker.zip](https://github.com/aws/sagemaker-python-sdk/files/6029177/sagemaker.zip)
> 
> If you run: `docker build -t sagemaker-test .`
> Then run: `docker run -it --rm sagemaker-test`
> 
> You will see the error.
> <img alt="image" width="590" src="https://user-images.githubusercontent.com/2687687/108851340-507e1200-75b2-11eb-94b4-2c6d7b97220f.png">

Thank you so much @carlowahlstedt for bringing attention to this issue.

I can confirm this is an issue again, as I was able to reproduce it. I'm not entirely sure why it is showing now.

I believe this will require some investigation on our end.

**Comment by marckarp** [comment]
Issue resolved/not reproducible with version 2.200.1 of the SDK
